packages/expo/static/loading-page/index.html

```diff
@@ -0,0 +1,274 @@
+          </defs>
```

Comment:
This is our template icon as an SVG with a much better size:
```suggestion
          <g>
            <rect width="1024" height="1024" rx="158" fill="#F8F8FA"/>
            <path d="M855 -1V1024" stroke="#E1E4E8" stroke-width="6"/>
            <path d="M684 -1V1024" stroke="#E1E4E8" stroke-width="6"/>
            <path d="M513 -1V1024" stroke="#E1E4E8" stroke-width="6"/>
            <path d="M342 -1V1024" stroke="#E1E4E8" stroke-width="6"/>
            <path d="M171 -1V1024" stroke="#E1E4E8" stroke-width="6"/>
            <path d="M1024 854L-1 854" stroke="#E1E4E8" stroke-width="6"/>
            <path d="M1024 683L-1 683" stroke="#E1E4E8" stroke-width="6"/>
            <path d="M1024 512L-1 512" stroke="#E1E4E8" stroke-width="6"/>
            <path d="M1024 341L-1 341" stroke="#E1E4E8" stroke-width="6"/>
            <path d="M1024 170L-1 170" stroke="#E1E4E8" stroke-width="6"/>
            <circle cx="512" cy="511" r="337" stroke="#D1D5DA" stroke-width="12"/>
            <circle cx="512" cy="512" r="166" stroke="#D1D5DA" stroke-width="12"/>
            <circle cx="512" cy="512" r="438" stroke="#D1D5DA" stroke-width="12"/>
          </g>
```